.infra/kube/deployment.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: tusd
+  namespace: tus
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: tusd
+    spec:
+      containers:
+      - image: docker.io/kiloreux/tusd:latest
```

Comment:
We have an official docker image at https://hub.docker.com/r/tusproject/tusd/ but I am not sure if it's always the latest master.

Reply:
Already aware of that one like already mentioned I don't have permissions there.